src/main/java/org/codeforamerica/shiba/output/caf/GrossMonthlyIncomePreparer.java

```diff
@@ -38,6 +41,9 @@ public GrossMonthlyIncomePreparer(GrossMonthlyIncomeParser grossMonthlyIncomePar
     List<JobIncomeInformation> jobsToIncludeInGrossIncome =
         getJobIncomeInformationToIncludeInThisDocument(application, document);
 
+    // Scope for self-employment and non self-employment jobs
+    AtomicInteger selfEmploymentIndex = new AtomicInteger(0);
+    AtomicInteger nonSelfEmploymentIndex = new AtomicInteger(0);
```

Comment:
I don't think we need `AtomicInteger`s, since there is no multithreading here. Can we just turn this into a regular loop with some `int`s?